### ISSUE TITLE
Issue desc [ICE AND FIRE VERSION]

### ISSUE THREAD

**Issue by SheroxGG** [Alex-the-666/Ice_and_Fire#3768]
**Describe the bug**
Some Dragons are not spawning on their caves. I am not sure if this has something to do with despawning, but some players on my server are reporting that they have found empty caves, Ive checked by myself and its true. No idea of why this is happening. Maybe it could be because despawn is activated and when someone load the cave and move away the dragon despawn forever?
I really need to fix this as soon as possible. Hope you can help! Thanks a lot for your time 

**To Reproduce**
Modded server, with some other mods and multiworld plugin

**Expected behavior**
Dragons spawning

**Screenshots**
Basically a empty cave roost without any player evidence

**Comment by TheBv** [comment]
Most likely fixed with #4245
